io-engine/src/core/env.rs

```diff
@@ -245,18 +245,19 @@ pub struct MayastorCliArgs {
         hide = true
     )]
     pub enable_nexus_channel_debug: bool,
+    #[clap(long, env = "LVM")]
+    pub lvm: bool,
```

Comment:
Should we call this `ENABLE_LVM` ?

Reply:
Also, could you add a comment description?

Reply:
Done